app/views/layouts/to_change_everything.html.erb

```diff
@@ -76,6 +76,7 @@
   <meta property="og:image" content="https://cloudfront.crimethinc.com/assets/tce/images/twitter-card-1486.jpg" />
   <meta property="og:type" content="article" />
 
+  <!-- TBD: fix path -->
   <link rel="stylesheet" href="https://cloudfront.crimethinc.com/tce/images/546676/CE039F8C7119876F1.css" />
```

Comment:
@veganstraightedge I cannot find this file under the `https://cloudfront.crimethinc.com/assets/tce/css/tcefont` path

Reply:
Copied it over.

`https://cloudfront.crimethinc.com/assets/tce/images/546676`…